client/app/previous-pres/previousPres.html

```diff
@@ -20,4 +20,22 @@
 	  	</div>
   	</div>
   </div>
+  <div class="row">
+    <div class="col-md-12">
+      <br>
+      <h1>Comments</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-7">
+      <h2>All Comments</h2>
+      <div class="comments">
+        <div class="comment" ng-repeat="comment in comments track by $index">
+          {{comment}}
+        </div>
+      </div>
+    </div>
+    <br>
+    <div id="wordcloud" class="col-md-3"></div>
+  </div>
```

Comment:
probably fine for now, but maybe include notes about why total columns are not 12 in total. Might consider adding col sizes for smaller pages to troubleshoot the issue (12 col's not looking right). If it works, though, just adding a comment makes more sense in terms of priority work for the moment.